src/app/theme/pages/default/features/users/user-add-edit/user-add-edit.component.ts

```diff
@@ -47,22 +48,46 @@ export class UserAddEditComponent implements OnInit {
     this.roleList = [];
     this.relatedSchoolList = [];
     this.currentUser = JSON.parse(localStorage.getItem('currentUser'));
-    if (this.currentUser && this.currentUser.roles && this.currentUser.roles.length > 0) {
-      this.userRole = this.currentUser.roles[0].name;
-    }
+    this.roleService.getAllRoles().subscribe(res => { 
+      if(res.length > 0){
+       // this.userRole = res.map(item => item.mstRole.roleName );
+      //  this.userRole = res[0].mstRole.roleName;
+       this.roleList = res.map(item => item );
+         console.log('this.roleList', this.roleList);
+      }
+  });
+    //if (this.currentUser && this.currentUser.roles && this.currentUser.roles.length > 0) {
+      //this.userRole = this.currentUser.roles[0].name;
+     
+    //}
 
     this.route.params.forEach((params: Params) => {
       this.params = params['userId'];
     });
-    if (this.userRole != 'SuperAdmin') {
-      this.getRolesBySchoolId();
+    
+    if (this.userRole == 'Administrator') {
+     // this.userService.getAllUsers().subscribe(res => { 
+      //if(res.length > 0){
+       // this.userRole = res.map(item => item.mstRole.roleName );
+      // this.roleList = res.map(item => item.mstRole );
+      //   console.log('this.roleList', this.roleList);
+     // }
+  //});
+      //this.roleList = this.userRole;
     } else {
       this.roleList = [{
         id: 1,
-        displayName: 'Super Admin'
+        roleName: 'Super Admin'
```

Comment:
Why this hard coded value for role list Update its with API 
http://localhost:50273/api/Role